src/wmnds/components/form-elements/textarea/_properties.njk.json

```diff
@@ -0,0 +1,122 @@
+{
+  "wmndsTextareaProps": [
+    {
+      "name": "id",
+      "type": "string",
+      "description": "<strong>Required.</strong> Must be unique. The id of the textarea."
+    },
+    {
+      "name": "name",
+      "type": "string",
+      "description": "<strong>Required.</strong> The name of the textarea, which is submitted with the form data."
+    },
+    {
+      "name": "rows",
+      "type": "string",
```

Comment:
Should this be an int?

Reply:
GDS has it as a string so I guess thats why Catia put it that way